### PR TITLE
DOCS: fix broken link

### LINKED ISSUE #1193: The link from cql_optimization to the prepared statements in scylla doc is broken
http://scylladb.github.io/scylla-monitoring/scylla-monitoring-3.5.2/cql_optimization.html#prepared-statements

The link in the section should link to scylla's documentation and it is broken

### PR DESCRIPTION
**Pull request by lauranovich**
FIX #1193

### CODE REVIEW

File: docs/source/cql_optimization.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Prepared Statements
 `Prepared statements`_ are queries that are first defined as a template with place holders for the values and then that template is used
 multiple times with different values.
 
-.. _`Prepared statements`: /getting-started/definitions/#prepared-statements
+.. _`Prepared statements`: http://docs.scylladb.com/getting-started/definitions/#prepared-statements
```
Comment (tzach) [inline docs/source/cql_optimization.rst]: please always use https

Comment (lauranovich) [inline docs/source/cql_optimization.rst]:
```suggestion
.. _`Prepared statements`: https://docs.scylladb.com/getting-started/definitions/#prepared-statements
```

**Review by tzach — APPROVED** [review]
LGTM

**Comment by lauranovich** [comment]
@amnonh - this should be backported to previous versions.

**Pull request merged**